compose/src/main/java/com/bottlerocketstudios/compose/auth/AuthCode.kt

```diff
@@ -63,6 +63,8 @@ fun AuthCodePreview() {
     }
 }
 
+// FIX ME: javaScriptEnabled = true can result in removal from Play Store
```

Comment:
@MMurrayAtBR nit: Change `FIX ME` `FIXME` to follow the naming used elsewhere in the project. Additionally, using `FIXME` allows project-wide searches to show all instances as well as cause all instances to show in the TODO view of Android Studio.

```suggestion
// FIXME: javaScriptEnabled = true can result in removal from Play Store
```